apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,29 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float minPulsewidthInMs, float maxPulsewidthInMs)
+    : m_servoPwmOut(servoPin), m_servoRangeInDegrees(servoRangeInDegrees), m_minPulsewidthInMs(minPulsewidthInMs), m_maxPulsewidthInMs(maxPulsewidthInMs) 
+    {}
+       
+// Set servo position (ex. 45 deg)
+void TutorialServo::setPositionInDegrees(const float degrees) {
+    m_servoPwmOut.period_ms(20);
+    m_servoPwmOut.pulsewidth_ms(m_minPulsewidthInMs + degrees/m_servoRangeInDegrees);
```

Comment:
Think about what happens if m_maxPulsewidthInMs changes

Reply:
Also think about if degrees was negative or greater than m_servoRangeInDegrees, how will your function handle them?

Reply:
pulsewidth_ms() takes an int as argument instead of a float, you'll want to use pulsewidth() instead and convert to seconds. Check out the Mbed documentation for details about their PwmOut API functions
https://os.mbed.com/docs/mbed-os/v6.10/mbed-os-api-doxy/classmbed_1_1_pwm_out.html#a7bc30e2d1f1c1ea873af9f57e19137a2